src/Sylius/Component/Shipping/Model/Shipment.php

```diff
@@ -165,7 +165,7 @@ public function getShippingUnitCount(): int
 
     public function getShippingUnitTotal(): int
     {
-        return 0;
+        throw new \Exception('You have to override and implement getShippingUnitTotal method');
```

Comment:
I'm not sure about this particular line. Maybe we should return `0` as it was before, but before that add deprecation error?

Reply:
I would revert it.
Those who use the shipping component standalone, either care about it and already overrode the method, or don't care about it and won't be affected anyway.
And for Sylius Standard you already provided an implementation in the core model.